sendgrid.go

```diff
@@ -55,6 +69,18 @@ func NewSendClient(key string) *Client {
 	return &Client{request}
 }
 
+// NewSendMultiClient constructs a new SendGrid client given an API key
+func NewSendMultiClient(keys []string) []*Client {
+	requests := GetRequests(keys, "/v3/mail/send", "")
```

Comment:
Would you add `/v3/mail/send` to the `const` above?

Reply:
sorry, what do you mean?

Reply:
@alimasyhur,

I believe what @robertacosta is saying is that you can set the string "/v3/mail/send" to be a constant variable.

Reply:
ok, will change it